content/en/agent/vector_aggregation.md

```diff
@@ -0,0 +1,149 @@
+---
+title: Aggregating multiple agents using Vector
+kind: documentation
+further_reading:
+- link: "/logs/"
+  tag: "Documentation"
+  text: "Collect your logs"
+- link: "/agent/proxy/"
+  tag: "Documentation"
+  text: "Configure the Agent to use a proxy"
+
+---
+
+## Overview 
+
+The Datadog agent can be used along with [Vector][1], in this scenario Agents will send data
+to Vector that will aggregate data from multiple incoming Agent:
+
+`Agents ---> Vector ---> Datadog`
+
+This scenario differs from using a [proxy][2] in the extend that Vector is able to
+process data before sending them to Datadog. Vector capabilities include, among others:
+* Log sampling
+* Log deduplication to avoid duplicated log to be send
+* Transforming logs using the [Vector Remap Language][3]
+
+
+**Note**: Only logs aggregation is supported, support for other data type (metrics, processes, traces, etc.)
+will be available later.
+
+**Note**: Vector can also directly collect logs from alternative sources, if doing so it is likely that those 
+third party logs won't come with proper tagging. A convenient way of [adding tags][4], source or service values is to do so using the [Vector Remap Language][3].
+
+## Configuration
+
+### Agent configuration
+To send logs to vector the Agent configuration file, `datadog.yaml` should be update to send data to Vector.
+As only logs data type are currently supported, you need to update the following values in your `datadog.yaml`:
+```yaml
+logs_config:
+  logs_dd_url: "<VECTOR_HOST>:<VECTOR_PORT>"
+  logs_no_ssl: true # If TLS/SSL is not enabled on the vector side
+```
+
+### Vector configuration
+To receive logs from Datadog Agent Vector should be configured with a [datadog_logs source][5].
+To send logs to Datadog, Vector should be configured with at least one [datadog_logs sink][6].
+
+Please check the official [Vector documentation][7] for all available configuration parameters and
+all transforms that can be applied to logs while they are processed by Vector.
+
+Sample configuration that simply add a tag to every log using the Vector Remap Language:
+```yaml
+sources:
+  datadog_agents:
+    type: datadog_logs
+    address: "[::]:8080"
+
+transforms:
+  add_tags:
+    type: remap
+    inputs:
+      - datadog_agents
+    source: |
+      .ddtags = add_datadog_tags!(.ddtags, ["sender:vector"])
+
+sinks:
+  to_datadog:
+    type: datadog_logs
+    inputs:
+       - add_tags
+    default_api_key: "${DATADOG_API_KEY_ENV_VAR}"
+    compression: gzip
+    encoding:
+      codec: json
+    healthcheck:
+      enabled: true
+```
+
+### Using Kubernetes
+Using the official Datadog chart the following values need to be updated:
```

Comment:
```suggestion

Using the official Datadog Helm chart, update the following values:

```

Reply:
cc @prognant I don't think this is actually resolved.